cognite/client/_api/extractionpipelines.py

```diff
@@ -202,6 +204,96 @@ def update(
             items=item,
         )
 
+    def get_config(
```

Comment:
A note on the exposed interface:
It would be more in line with the rest of the SDK if these were in a separate class made available through an attribute on `ExtractionPipelinesAPI`, so you would get 

`client.extractionpipelines.get_config` -> `client.extractionpipelines.config.retrieve`
`client.extractionpipelines.list_config_revisions` -> `client.extractionpipelines.config.list_revisions`
`client.extractionpipelines.new_config` -> `client.extractionpipelines.config.create`
`client.extractionpipelines.revert_config` -> `client.extractionpipelines.config.revert`

Take a look at e.g. `client.sequences.data` or `client.transformations.schedules`

What do you think?

Reply:
That makes sense. I'll have a look.

Reply:
I changed this, and added a reference to `ExtractionPipelineRunsAPI` as well, though we still have `client.extraction_pipeline_runs`, removing that would break backwards compatibility.